plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/connection/DockerConnection.java

```diff
@@ -61,6 +62,11 @@ public DockerConnection query(String name, Object... values) {
         return this;
     }
 
+    public DockerConnection header(String key, Object value) {
+        this.headers.add(Pair.of(key, value));
+        return this;
+    }
+
     public DockerConnection headers(List<Pair<String, ?>> headers) {
         this.headers = headers;
```

Comment:
I think that your changes introduced a bug. What if argument of this method is immutable list and method `header(key, value)` will be called after this method. In that case you will get exception